Whiskey/Tasks/Set-WhiskeyProGetAsset.ps1

```diff
@@ -0,0 +1,97 @@
+
+
+function Set-WhiskeyProGetAsset {
+    <#
+        .SYNOPSIS
+        Adds and Updates assets to the ProGet asset manager. 

```

Comment:
Please user proper casing in documentation. `Updates` should be lower-cased: `updates`.

Reply:
Done